Week1/homework/nadin.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+let greetingIndonesian = 'Ciao';
+let greetingArabic = 'مرحبا ';
+let greetingSpanish = 'Halo';
+ console.log(greetingIndonesian, greetingArabic, greetingSpanish);
+
+
+console.log("I'm awesome");
+
+
+let x;
+console.log("The value of my variable number will be: undefined");
+console.log(x);
+x = 1;
+console.log("The value of my variable number will be: 1");
+console.log(x);
+
+let y = "the first string";
+console.log("the value of my variable y will be: the first string");
+console.log(y);
+y = "the second string";
+console.log("the value of my variable string1 will be: the second string");
+console.log(y);
+
+console.log(Math.round(7.25));
+// expected output: 8
```

Comment:
Are you sure this is going to be printed as 8? Can you please check? Also, I would like if you tried to find out the difference between `Math.floor` and `Math.round` 🙂

Reply:
It 's 7 , now I am sure! :) That helps me well! Thanks Yash👍